README.md

```diff
@@ -192,6 +192,8 @@ Created by Andreas on June 17th, 2021. Reason: unknown. Various people used the
 
 There's a yak hiding here. When something seems to lead to a shaveable yak, but you currently only see its traces.
 
+Alternate meaning: baby steps, but with yaks.
+
```

Comment:
As this seems to be the original meaning, it appears that I have misunderstood this emoji. You can remove the currently present meaning and use this one as the "default", and mention my "alternate" meaning as the alternative that it is.

Reply:
Done